manifests/s/SimonG4ELI/SDR-Radio/3.3-b3195/SimonG4ELI.SDR-Radio.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SimonG4ELI.SDR-Radio
+PackageVersion: 3.3-b3195
+PackageLocale: en-US
+Publisher: Simon G4ELI
+PackageName: SDR-Radio
```

Comment:
```suggestion
PublisherUrl: https://github.com/SimonG4ELI
PackageName: SDR-Radio
PackageUrl: https://www.sdr-radio.com/download#Release
```

Reply:
Hi @AsciiWolf,

I tracked down the publisher's website and PackageUrl.

Reply:
Thanks! :-)